### PR TITLE
Add Roadie product info to landing page

### PR DESCRIPTION
**Pull request by dtuite**
[ch3215]

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -112,20 +102,26 @@ const Home = ({ data, location }) => {
...
             text={
               <div>
-                <h2 className={classes.h2}>Custom plugins</h2>
-                <p className={classes.p}>Ready to roll, easy to extend.</p>
+                <h2 className={classes.h2}>Maintenance and Security built in</h2>
+                <p className={classes.p}>Backstage moves quickly and it takes effort to stay up to date.</p>
                 <p className={classes.p}>
-                  Common SaaS tools come baked in. When you need to break out you can roll your own
-                  and deploy with a simple push.
+                  Enter some details about your custom plugin into Roadie, copy some credentials, then `npm publish` into our private and secure pipeline. Slack notifications included.
```
Comment (martina-if) [inline src/pages/index.js:111]: This line seems copy paste from line 78

Comment (dtuite) [reply src/pages/index.js]: Yep. I was just copy/pasting text to fill out the visuals. It's fixed up now.

File: src/components/SitewideHeader/index.js [hunk 2]
```diff
@@ -29,6 +29,18 @@ const SitewideHeader = ({
 
           <nav className={classes.fullScreenNav} aria-label="Sitewide navigation">
             <span>
+              {['lg', 'xl', 'none'].includes(maxWidthBreakpoint) && (
```
Comment (dtuite) [inline src/components/SitewideHeader/index.js:32]: Without this, there are too many links in the header on small screen sizes which are not small enough to be mobile width.

File: src/components/ResponsiveSpacer.js [hunk 3]
```diff
@@ -2,10 +2,17 @@ import React from 'react';
 import { createUseStyles } from 'react-jss';
 import classnames from 'classnames';
 
-const useStyles = createUseStyles(() => ({
+const useStyles = createUseStyles((theme) => ({
   root: {
-    paddingTop: 60,
-    paddingBottom: 60,
+    paddingTop: 24,
```
Comment (dtuite) [inline src/components/ResponsiveSpacer.js:7]: This is unrelated but I noticed the vertical padding was a bit excessive on mobiles.

**Review by Xantier — APPROVED** [review]
LGTM. An alternative to try some time in the future could be alternative side-by-side for those features like this TailwindUI example does:
image

**Comment by dtuite** [comment]
@Xantier Yeah I had a version like that and it looked weird for some reason I couldn't totally understand. I can throw up a version like that for us to look at. I'll get this released first though.

**Pull request merged**